src/hooks/useSortingDropdowns.ts

```diff
@@ -0,0 +1,136 @@
+import { SortType } from '@/enums';
+import { useSearchParams } from 'react-router-dom';
+
+export const useSortingDropdowns = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sortingParams = [
+    { value: SortType.NONE, label: 'None' },
+    { value: SortType.NEWEST, label: 'Newest' },
+    { value: SortType.OLDEST, label: 'Oldest' },
+    { value: SortType.PRICE_HIGH, label: 'Price high' },
+    { value: SortType.PRICE_LOW, label: 'Price low' },
+  ];
```

Comment:
Good idea to create a hook, but I don't know what will be with our previous code, because there I used library for translations.😳